### DISCUSSION TITLE
Enabling h-extension

### DISCUSSION THREAD

**Discussion by FrancoAuri96** [chipsalliance/rocket-chip#3503] [Q&A]
Hello,
I'd like to know how to enable the h-extension during the rocket chip generation.
Could anyone help me?

**Comment by sequencer** [comment]
Try to generate emulator in the `dev` branch with:
`nix develop -c mill "emulator[freechips.rocketchip.system.TestHarness,freechips.rocketchip.system.HypervisorConfig].elf`

**Comment by FrancoAuri96** [comment]
ok i'll try it, thank you!
